sleuth_backend/solr/models.py

```diff
@@ -69,8 +69,26 @@ class CourseItem(SolrDocument):
         "updatedAt": "",
         "description": "",
         "subjectId": "",
-        "subjectData": []
```

Comment:
It might be better to turn this into an object with key-value pairs.

Reply:
will change to multiple fields